packages/terra-hyperlink/src/terra-dev-site/doc/hyperlink/Hyperlink.1.doc.mdx

```diff
@@ -0,0 +1,55 @@
+import { Badge } from 'terra-hyperlink/package.json?dev-site-package';
+
+import DefaultHyperlink from '../example/DefaultHyperlink?dev-site-example';
+import Audio from '../example/AudioHyperlink?dev-site-example';
+import Disabled from '../example/DisabledHyperlink?dev-site-example';
+import Document from '../example/DocumentHyperlink?dev-site-example';
+import External from '../example/ExternalHyperlink?dev-site-example';
+import Image from '../example/ImageHyperlink?dev-site-example';
+import UnderlineHidden from '../example/UnderlineHiddenHyperlink?dev-site-example';
+import Video from '../example/VideoHyperlink?dev-site-example';
+
+import HyperlinkPropsTable from 'terra-hyperlink/src/Hyperlink?dev-site-props-table';
+
+<Badge />
+
+# Terra Hyperlink
+
+The terra hyperlink component allows linking to other web pages, files, locations within the same page, email addresses, or any other URL.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-hyperlink`
+
+<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
+## Peer Dependencies
+
+This component requires the following peer dependencies be installed in your app for the component to properly function.
+
+| Peer Dependency | Version |
+|-|-|
+| react | ^16.8.5 |
+| react-dom | ^16.8.5 |
+
+<!-- AUTO-GENERATED-CONTENT:END -->
+
+## Component Features
+
+ * [Cross-Browser Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#cross-browser-support)
+ * [LTR/RTL Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#ltr--rtl-support)
+
+[View component source code](https://github.com/cerner/terra-core/tree/master/packages/terra-hyperlink)
```

Comment:
This is missing the usage section for quick reference for the component import

Reply:
Oops, I merged this before i saw this comment. @pranav300 could you open another pr to address this? (sorry 🙁)